flatflow/torch/nn/modules/attention.h

```diff
@@ -0,0 +1,40 @@
+#include <cublas_v2.h>
+#inlcude <iostream>
+
+namespace flatflow {
+namespace torch {
+namespace nn {
+namespace modules {
+
+// flatflow::kernel::cuda_check
+//
+// utility function to check CUDA errors
+inline void cuda_check(cudaError_t error, const char *file, int line) {
+  if (error != cudaSuccess) {
+    std::cerr << "CUDA error at " << file << ":" << line << " : "
+              << cudaGetErrorString(error) << std::endl;
+    exit(EXIT_FAILURE);
+  }
+};
+
+// flatflow::kernel::cublas_check
+//
+// utility function to check cuBLAS status errors
+inline void cublas_check(cublasStatus_t status, const char *file, int line) {
+  if (status != CUBLAS_STATUS_SUCCESS) {
+    std::cerr << "cuBLAS error at " << file << ":" << line << " : "
+              << status << std::endl;
+    exit(EXIT_FAILURE);
+  }
+}
+
+// flatflow::kernel::split_attention
+//
+// Split input tensor into multiple tensors based on the split indices
+std::vector<torch::Tensor> split_attention(
+    torch::Tensor &QKV, const std::vector<int> &split_indices);
```

Comment:
I think ``split_indices`` should be a ``const std::vector<std::size_t> &``.
Accordingly, code that uses ``split_indices`` should also use ``std::size_t``.